lib/inferno/apps/cli/evaluate.rb

```diff
@@ -2,13 +2,58 @@
 require_relative '../../dsl/fhir_evaluation/config'
 require_relative '../../entities'
 require_relative '../../utils/ig_downloader'
+require_relative 'migration'
 
 require 'tempfile'
+require 'faraday'
 
 module Inferno
   module CLI
-    class Evaluate < Thor::Group
-      def evaluate(ig_path, data_path, _log_level)
+    class Evaluate
+      # @see Inferno::CLI::Main#evaluate
+      def run(ig_path, data_path, options)
+        tmpdir = Dir.mktmpdir
+        Dir.mkdir("#{tmpdir}/data")
+        Dir.mkdir("#{tmpdir}/data/igs")
+        ENV['TMPDIR'] = tmpdir
+        ENV['FHIRPATH_URL'] = 'http://localhost:6789'
+        ENV['FHIR_RESOURCE_VALIDATOR_URL'] = 'http://localhost:3500'
+
+        puts 'Starting Inferno Evaluator Services...'
+        system("#{services_base_command} up -d #{services_names}")
+
+        ig_path = if ig_path.starts_with? '~'
+                    ig_path.sub('~', Dir.home)
+                  else
+                    File.absolute_path(ig_path)
+                  end
+
+        Dir.chdir(tmpdir) do
+          Migration.new.run(Logger::FATAL) # Hide migration output for evaluator
+          evaluate(ig_path, data_path, options)
```

Comment:
Won't `data_path` need to be updated to an absolute path prior to changing the working directory?

Reply:
fixed